test/distributed/pipelining/test_stage.py

```diff
@@ -0,0 +1,200 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates
+# Owner(s): ["oncall: distributed"]
+import copy
+import os
+import sys
+import tempfile
+
+import torch
+import torch.distributed as dist
+
+from model_registry import ExampleCode, MultiMLP, ModelWithKwargs, MLPModule
+from torch.distributed.pipelining import (
+    pipeline,
+    ManualPipelineStage,
+    PipelineStage,
+    Schedule1F1B,
+    ScheduleGPipe,
+)
+from torch.testing._internal.common_cuda import TEST_MULTIGPU
+from torch.testing._internal.common_distributed import (
+    MultiProcContinousTest,
+    requires_nccl,
+)
+from torch.testing._internal.common_utils import (
+    instantiate_parametrized_tests,
+    parametrize,
+    skip_but_pass_in_sandcastle_if,
+)
+
+
+d_hid = 512
+batch_size = 256
+chunks = 4
+
+torch.manual_seed(0)
+
+
+class StageTest(MultiProcContinousTest):
```

Comment:
Just curious, what does `MultiProcContinousTest` do? How is it different than `MultiProcessTestCase`